Libplanet/Store/Trie/Nodes/BaseNode.cs

```diff
@@ -21,8 +13,6 @@ protected BaseNode(INode? value)
         // It will not support embedded node.
         public INode? Value { get; }
 
-        public byte[] Serialize() => _codec.Encode(ToBencodex());
```

Comment:
IMO this change needs to describe on CHANGES.md

Reply:
I omitted it because `BaseNode` is an `internal class`.